tests/sqlite/query.rs

```diff
@@ -910,12 +910,12 @@ fn select_55() {
             r#"SELECT "aspect""#,
             r#"FROM "glyph""#,
             r#"WHERE IFNULL("aspect", 0) > 2"#,
-            r#"ORDER BY CASE"#,
+            r"ORDER BY CASE",
             r#"WHEN "id"=4 THEN 0"#,
             r#"WHEN "id"=5 THEN 1"#,
             r#"WHEN "id"=1 THEN 2"#,
             r#"WHEN "id"=3 THEN 3"#,
-            r#"ELSE 4 END,"#,
+            r"ELSE 4 END,",
             r#""glyph"."aspect" ASC"#,
         ]
         .join(" ")
```

Comment:
Thank you for the above changes. 
While I think most removal are fine when they're clearly not necessary, this bits look inconsistent with the surrounding lines?

Can you revert the changes where it is in a vec of string and the other lines still need `r#`?

Reply:
You're welcome. I reverted the changes in specific spots to make it more consistent.